packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx

```diff
@@ -228,8 +228,15 @@ export class CommandBar extends BaseComponent<ICommandBarProps, ICommandBarState
             aria-haspopup={ hasSubmenuItems(item) }
           >
             { (hasIcon) ? this._renderIcon(item) : (null) }
-            <span className={ css(
-              'ms-CommandBarItem-commandText', styles.itemCommandText) } aria-hidden='true' role='presentation'>{ item.name }</span>
+            { (!!item.name) && (
```

Comment:
@micahgodbolt If it's worth anything, I _really_ like the [Prettier formatter](https://github.com/prettier/prettier) and the opinions it has on formatting JS/TS/JSX/TSX. It has a VSCode extension that works very well.